### DISCUSSION TITLE
Major problems

### DISCUSSION THREAD

**Discussion by CapJambo** [flybywiresim/aircraft#5612] [General]
For some wierd reason I had the flybywire installed into the same folder as every other aircraft that comes with MSFS2020. That means I had going 2 copies of the same aircraft. No wonder I had major problems since the instaler came out.
Problems like:
CTD
Flightbag white or displaying a wrong version
Landing, strobes, beacon and runway switches did not work at all. 

Peace

**Comment by Alexis241168** [comment]
Hola 
Tengo el mismo problema con las luces, además no me permite crear mi propio plan de vuelo en el FMC.
